e2e/features/social/share-test.js

```diff
@@ -0,0 +1,44 @@
+const reuseables = require('../../reuseables/skip-tour.js');
+const localSelectors = require('../../reuseables/selectors.js');
+
+const TIME_LIMIT = 10000;
+
+const {
+  socialToolbar,
+  socialCopyLinkButton,
+  socialLinkInput,
+} = localSelectors;
+
+module.exports = {
+  before(client) {
+    reuseables.loadAndSkipTour(client, TIME_LIMIT);
+  },
+  'Clicking the social link button opens the social share dialog': (client) => {
+    client.waitForElementVisible(socialCopyLinkButton, TIME_LIMIT);
+    client.click(socialCopyLinkButton);
+    client.pause(1000);
```

Comment:
Is there a particular reason you chose to `pause()` for a second here?

I wonder if it would be quicker/more reliable to use `waitForElementVisible(socialToolbar)`?

Reply:
It must have been early test animation related that I did, but `waitForElementVisible` works better so I'll make that change.